apps/showcase/src/assets/trainings/sdk/steps/angular-integration/exercise/app.component.ts

```diff
@@ -0,0 +1,34 @@
+import { JsonPipe } from '@angular/common';
+import { Component, inject, signal } from '@angular/core';
+import { ApiFactoryService } from '@o3r/apis-manager';
+import { type Pet, PetApi, StoreApi } from 'sdk';
+
+@Component({
+  selector: 'app-root',
+  standalone: true,
+  imports: [JsonPipe],
+  templateUrl: './app.component.html',
+  styleUrl: './app.component.scss'
+})
+export class AppComponent {
+  /** Title of the application */
+  public title = 'tutorial-app';
+
+  /* Inject the ApiFactoryService and get the corresponding APIs */
+  private readonly petApi;
+  private readonly storeApi;
+
+  private readonly petsWritable = signal<Pet[]>([]);
+  public readonly pets = this.petsWritable.asReadonly();
+
+  private readonly petsInventoryWritable = signal<{ [key: string]: number }>({});
+  public readonly petsInventory = this.petsInventoryWritable.asReadonly();
+
+  public async getAvailablePets() {
+    /* Get the pets whose status is 'available' */
+  }
+
+  public async getPetInventory() {
+    /* Get the pets inventory */
+  }
```

Comment:
Shouldn't this part already part of another training ?
here we expect him to inject the good Api L18-L19, no ?
```suggestion
  public async getAvailablePets() {
    const availablePets = await this.petApi.findPetsByStatus({status: 'available'});
    this.petsWritable.set(availablePets);
  }

  public async getPetInventory() {
    const inventory = await this.storeApi.getInventory({});
    this.petsInventoryWritable.set(inventory);
  }
```